docs/source/exposing.md

```diff
@@ -0,0 +1,266 @@
+
+![ClusterIPs](static/exposing/clusterip.svg)
+
+### In-cluster node-to-node, VPC clients-to-nodes
+
+ScyllaCluster definition:
+```yaml
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+spec:
+  exposeOptions:
+    nodeService:
+      type: ClusterIP
+    broadcastOptions:
+      clients:
+        type: PodIP
+      nodes:
+        type: ServiceClusterIP
+```
+
+In this scenario, we assume that the Pod IP subnet is routable within a VPC. 
+Clients within the VPC network can communicate directly with ScyllaCluster nodes using Pod IPs.
+Nodes communicate with each other exclusively within the same Kubernetes cluster.
+
+![PodIPs](static/exposing/podips.svg)
+
+### Multi Region
+
+ScyllaCluster definition:
+```yaml
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+spec:
+  exposeOptions:
+    nodeService:
+      type: Headless
+    broadcastOptions:
+      clients:
+        type: PodIP
+      nodes:
+        type: PodIP
+```
+
+In this scenario, we set up two separate Kubernetes clusters in different regions, each within its respective VPC. 
+These VPCs are interconnected to facilitate inter-VPC connectivity. 
+We operate on the assumption that the Pod IP subnet is routable within each VPC. 
+Nodes, deployed in these distinct regions, broadcast their Pod IP addresses, enabling them to establish connections with one another. 
+Clients, whether deployed within the same Kubernetes cluster or within a VPC, have the capability to reach nodes using their Pod IPs.
+
+Since there is no requirement for any address other than the Pod IP, the `Headless` service type is sufficient.
+
+
+![MultiDC](static/exposing/multidc.svg)
+
+### Internet
+
+ScyllaCluster definition:
+```yaml
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+spec:
+  exposeOptions:
+    nodeService:
+      type: LoadBalancer
+    broadcastOptions:
+      clients:
+        type: ServiceLoadBalancerIngress
+      nodes:
+        type: ClusterIP 
+```
+
+We assume that a Kubernetes cluster has been deployed in a cloud provider environment that supports external load balancers. 
+By specifying the LoadBalancer type in the nodeService template, the Scylla Operator generates a dedicated LB Service for each node. 
+The cloud provider then establishes an external load balancer with an internet-accessible address. 
+ScyllaDB nodes broadcast this external address to clients, enabling drivers to connect and discover other nodes. 
+Since all ScyllaDB nodes reside within the same Kubernetes cluster, there is no need to route traffic through the internet. 
+Consequently, the nodes are configured to communicate via ClusterIP, which is also accessible within LoadBalancer Services.
+
+![Internet](static/exposing/loadbalancer.svg)
+
+-- -
+
+Other more complex scenarios can be built upon these simple ones.
+If you have any more questions regarding it, we will welcome you on our [ScyllaDB Users Slack](http://slack.scylladb.com/), under `#scylla-operator` channel.
```

Comment:
```suggestion
Join the `#scylla-operator` Slack Channel in [ScyllaDB Users](http://slack.scylladb.com/) for comments and assistance related to exposing ScyllaClusters.
```
nit

Reply:
On the other hand, do we really want to have that in separate docs? We have a link pointing to the users slack in the footer?

Reply:
Removed